eagles-food-backend/appsettings.json

```diff
@@ -1,6 +1,6 @@
 {
   "ConnectionStrings": {
-    "DefaultConnection": "To be changed to prod db"
+    "DefaultConnection": "Server=MYSQL8002.site4now.net;Database=db_a9ef58_teagle;Uid=a9ef58_teagle;Pwd=c7b9@r.jDR7.ZKW"
   },
```

Comment:
please run `git update-index --assume-unchanged appsettings.json` and set your default strings in the Development one, since that's not monitored by git anymore